homework_3_objects/main.js

```diff
@@ -0,0 +1,79 @@
+// Duration between dates
+
+const durationBetweenDates = (startDate = '1 Jan 1970', endDate = Date(), units = 'days') => {
+  const start = new Date(startDate);
+  const end = new Date(endDate);
+
```

Comment:
Mess in naming `startDate` and `endDate` are strings, `start` and `end` are Dates

Reply:
Oh, I see it 🤦‍♀️ Changed names, thank you!